test/config/config.ts

```diff
@@ -1,98 +1,346 @@
-    host: process.env.DB2_PORT_50000_TCP_ADDR || process.env.SEQ_DB2_HOST || process.env.SEQ_HOST || process.env.IBM_DB_HOSTNAME || '127.0.0.1',
-    port: process.env.DB2_PORT_50000_TCP_PORT || process.env.SEQ_DB2_PORT || process.env.SEQ_PORT
-      || process.env.IBM_DB_PORT || 50_000,
-    pool: {
-      max: Number(process.env.SEQ_DB2_POOL_MAX || process.env.SEQ_POOL_MAX || 5),
-      idle: Number(process.env.SEQ_DB2_POOL_IDLE || process.env.SEQ_POOL_IDLE || 3000),
+    // DB2 v11.5.5 and older: 50000; v11.5.6 and newer: 25000
+    native: { port: 50_000 },
+    default: {
+      database: env.SEQ_DB2_DB || env.SEQ_DB || env.IBM_DB_DBNAME || 'testdb',
+      username: env.SEQ_DB2_USER || env.SEQ_USER || env.IBM_DB_UID || 'db2inst1',
+      password: env.SEQ_DB2_PW || env.SEQ_PW || env.IBM_DB_PWD || 'password',
+      host: env.DB2_PORT_50000_TCP_ADDR || env.SEQ_DB2_HOST || env.SEQ_HOST || env.IBM_DB_HOSTNAME || '127.0.0.1',
+      port: env.DB2_PORT_50000_TCP_PORT || env.SEQ_DB2_PORT || env.SEQ_PORT || env.IBM_DB_PORT, // || 50_000,
+      pool: {
+        max: Number(env.SEQ_DB2_POOL_MAX || env.SEQ_POOL_MAX || 5),
+        idle: Number(env.SEQ_DB2_POOL_IDLE || env.SEQ_POOL_IDLE || 3000),
+      },
+    },
+    11.5: {
+      port: env.DB2_PORT_50000_TCP_PORT || env.SEQ_DB2_PORT || env.SEQ_PORT || env.IBM_DB_PORT || 50_000,
     },
   },
+
+  //  ╦╔╗ ╔╦╗╦  ┌─┐┌─┐┌┬┐┌┬┐┬┌┐┌┌─┐┌─┐
+  //  ║╠╩╗║║║║  └─┐├┤  │  │ │││││ ┬└─┐
+  //  ╩╚═╝╩ ╩╩  └─┘└─┘ ┴  ┴ ┴┘└┘└─┘└─┘
+
   ibmi: {
-    database: env.SEQ_IBMI_DB || env.SEQ_DB,
-    username: process.env.SEQ_IBMI_USER || process.env.SEQ_USER,
-    password: process.env.SEQ_IBMI_PW || process.env.SEQ_PW,
-    pool: {
-      max: Number(env.SEQ_IBMI_POOL_MAX || env.SEQ_POOL_MAX || env.SEQ_POOL_MAX || 5),
-      idle: Number(env.SEQ_IBMI_POOL_IDLE || env.SEQ_POOL_IDLE || 3000),
-    },
-    dialectOptions: {
-      odbcConnectionString: env.SEQ_IBMI_CONN_STR,
+    native: { port: 8471 },
+    default: {
+      database: env.SEQ_IBMI_DB || env.SEQ_DB,
+      username: env.SEQ_IBMI_USER || env.SEQ_USER,
+      password: env.SEQ_IBMI_PW || env.SEQ_PW,
+      pool: {
+        max: Number(env.SEQ_IBMI_POOL_MAX || env.SEQ_POOL_MAX || env.SEQ_POOL_MAX || 5),
+        idle: Number(env.SEQ_IBMI_POOL_IDLE || env.SEQ_POOL_IDLE || 3000),
+      },
+      dialectOptions: {
+        odbcConnectionString: env.SEQ_IBMI_CONN_STR,
+      },
     },
   },
 };
+
+//   ██████╗ ██████╗ ███╗   ██╗███████╗██╗ ██████╗     ███████╗██╗   ██╗███╗   ██╗ ██████╗
+//  ██╔════╝██╔═══██╗████╗  ██║██╔════╝██║██╔════╝     ██╔════╝██║   ██║████╗  ██║██╔════╝
+//  ██║     ██║   ██║██╔██╗ ██║█████╗  ██║██║  ███╗    █████╗  ██║   ██║██╔██╗ ██║██║
+//  ██║     ██║   ██║██║╚██╗██║██╔══╝  ██║██║   ██║    ██╔══╝  ██║   ██║██║╚██╗██║██║
+//  ╚██████╗╚██████╔╝██║ ╚████║██║     ██║╚██████╔╝    ██║     ╚██████╔╝██║ ╚████║╚██████╗
+//   ╚═════╝ ╚═════╝ ╚═╝  ╚═══╝╚═╝     ╚═╝ ╚═════╝     ╚═╝      ╚═════╝ ╚═╝  ╚═══╝ ╚═════╝
+//
```

Comment:
That's just editor width. The text is 90-columns, you almost have to squish your editor to purposely wrap it like that, but who codes with word-wrapping?

I'm not married to this. Like you, I saw it done elsewhere and I had concerns about its accessibility.  I was concerned more with the seeing-impaired.  I later discovered it was a much more pleasurable experience when re-reading code or  reading other people's code -- like dramatically different.  For longer files, it helped break up the parts when scanning.  The big headers were even readable in editor sidebars, improving mouse-drag navigation. And screen readers can still read the function and their names; its additional, not mandatory.

Again, I'm not married to it, but there's more than meets the eye and want to make sure that's also considered.

Reply:
I often look at PRs from my phone, and there the length is different. So the text does not show properly. It does indeed have some benefits but not enough to be used in Sequelize in my opinion

Reply:
> but who codes with word-wrapping?

GitHub's split diff view uses word-wrapping

Consider using something like this if you need to split sections:

```
// ============================================
//  Special Cases
// ============================================
```

I've also seen people use `-` as a lower level of importance:

```
// --------------------------------------------
//  Special Cases - Part 1
// --------------------------------------------
```

I don't think the many = are going to be screen reader friendly either but the text inside of it can be read, and remains readable with word-wrapping